### ISSUE TITLE
Build Setup: Combining the Maven plugins 'maven-shade-plugin' and 'gradle-module-metadata-maven-plugin' in the afterburner pom breaks Gradle metadata

### ISSUE THREAD

**Issue by jjohannes** [FasterXML/jackson-modules-base#200]
Solutions could be:
- Somehow configure the afterburner build setup to add the Marker comment to the `dependency-reduced-pom.xml`
- Fix up the 'gradle-module-metadata-maven-plugin' plugin to somehow deal with the `dependency-reduced-pom.xml` from the shade plugin automatically. As it is generated, maybe the comment can also be generated, and then the user of the plugins won't have to take care of it.

**Comment by jjohannes** [comment]
Solution to fix it in the build. Add this to the end of the `<plugins>` block `afterburner/pom.xml`. It adds the comment to the generated pom.

```
      <plugin>
        <groupId>io.github.floverfelt</groupId>
        <artifactId>find-and-replace-maven-plugin</artifactId>
        <version>1.1.0</version>
        <executions>
          <execution>
            <id>exec</id>
            <phase>package</phase>
            <goals>
              <goal>find-and-replace</goal>
            </goals>
            <configuration>
              <replacementType>file-contents</replacementType>
              <baseDir>${basedir}</baseDir>
              <findRegex><![CDATA[<modelVersion>4.0.0</modelVersion>]]></findRegex>
              <fileMask>dependency-reduced-pom.xml</fileMask>
              <replaceValue><![CDATA[
                <!-- do_not_remove: published-with-gradle-metadata -->
                <modelVersion>4.0.0</modelVersion>
              ]]></replaceValue>
              <recursive>false</recursive>
            </configuration>
          </execution>
        </executions>
      </plugin>
```

**Comment by jjohannes** [comment]
Fix above has been integrated via #198
